slack-plugin/src/main/kotlin/slack/gradle/SlackGradleUtil.kt

```diff
@@ -242,13 +242,13 @@ internal fun VersionCatalog.identifierMap(): Map<String, String> {
  * nesting.
  */
 internal fun tomlKey(key: String): String =
-  key.replace("-", "%").replace(".", "-").replace("%", ".").replace("_", ".").toCamel()
+  key.replace("-", "%").replace(".", "-").replace("%", ".").replace("_", ".").snakeToCamel()
 
-private fun String.toCamel(): String {
+internal fun String.snakeToCamel(upper: Boolean = false): String {
```

Comment:
ported the updated version from vulcan. Ended up not being needed here but leaving in as a toe-hold for future stuff